script.js

```diff
@@ -20,76 +20,16 @@ document.querySelectorAll('.nav-link').forEach((n) => n.addEventListener('click'
   back.classList.remove('active');
 }));
 
-// form validation
-const form = document.querySelector('form');// get the form
-const username = document.getElementById('name');// get the name
-const email = document.getElementById('email');// get the email
-const message = document.getElementById('message');// get the message
-const errorDisplay = document.querySelector('.errorMsg');
-// for setting the error message
-const setError = (element, message) => {
-  errorDisplay.innerText = message;
-  errorDisplay.classList.add('error');
-  errorDisplay.classList.remove('success');
-};
-
-// for setting the the success message
-const setSuccess = () => {
-  errorDisplay.innerText = '';
-  errorDisplay.classList.add('success');
-  errorDisplay.classList.remove('error');
-};
-
-let submitPt = 0;// for calcuating the result should be 3
-
-const validateInputs = () => {
-  const usernameValue = username.value.trim();
-  const emailValue = email.value.trim();
-  const messageValue = message.value.trim();
-
-  // for name validation
-  if (usernameValue === '') {
-    setError(username, 'Username is required');
-    return;
-  } if (usernameValue.split('').length > 30) {
-    setError(username, 'Username should not exceed 30 characters');
-    return;
-  }
-  setSuccess(username);
-  submitPt += 1;
-
-  // for email validation
-  if (emailValue === '') {
-    setError(email, 'Email is required');
-    return;
-  } if (emailValue !== emailValue.toLowerCase()) {
-    setError(email, 'Email should be in lowercase');
-    return;
-  }
-  setSuccess(email);
-  submitPt += 1;
-
-  // for message validation
-  if (messageValue === '') {
-    setError(message, 'Please add a message');
-  } else if (messageValue.split('').length > 500) {
-    setError(message, 'Message should be short and do not exceed 500 characters');
-  } else {
-    setSuccess(message);
-    submitPt += 1;
-  }
-};
-
-// the validation when the form is submitted
-form.addEventListener('submit', (e) => {
-  e.preventDefault();// prevent sumitting
-
-  validateInputs();// run checks
-
-  // deciding to submit or not
-  if (submitPt === 3) {
-    form.submit();
+const formbutton = document.getElementById('bot');
+const Email = document.getElementById('email');
+const Errormessage = document.getElementById('error-message');
+Errormessage.textContent = 'Message not sent! Please write using only the small letters. Thank you.';
```

Comment:
- [ ] You did a good job on the functionalities but there is a little issue, Please include email in your error message so users will get to know that they need to type the email in lower case. It will improve the user experience of your website.

Reply:
@thecodechaser  how should i do that  can you write it up so that I would understand

Reply:
@cynthia-nwannah you can print something like this `"Message not sent! Please write the email using only the small letters. Thank you."`